src/main/java/com/chessmates/league/LeagueTable.groovy

```diff
@@ -0,0 +1,70 @@
+package com.chessmates.league
+
+import com.chessmates.utility.GameColor
+import org.slf4j.Logger
+
+/**
+ * A simple league table.
+ */
+class LeagueTable {
+
+    private static final Logger logger = org.slf4j.LoggerFactory.getLogger(LeagueTable)
+
+    final winPoints
+    final loosePoints
```

Comment:
Loose points are the amount of points given for a loss. May not make sense for a normal points system (1 for win, 0 for loss) but since we're providing these values as a constructor you could conceivably give some value to a lost game too. Will add a comment.